recipe/pixi.toml

```
@@ -0,0 +1,26 @@
+[project]
+name = "repodata-patches"
+channels = ["conda-forge"]
+platforms = ["osx-arm64", "linux-64", "win-64"]
+
+[tasks.diff]
+cmd = "python show_diff.py {{ arguments }} 1> show_diff_result.txt"
+description = "Slow!! Writes repodata changes to file, add this result to the PR."
+args = [{arg = "arguments", default = ""}]
+
+[tasks.diff-cached]
+depends-on = [{task = "show-diff", args = ["--use-cache"]}]
+
+[tasks.lint]
+cmd = "pre-commit run --all-files"
+description = "Run pre-commit hooks to lint the code."
+
+[tasks.timestamp]
+cmd = "python -c \"import time; print(f'{time.time():.0f}000')\""
+description = "Generate a timestamp in milliseconds since epoch."
```

If we make this a task, it should be timezone-aware. Our infra operates on UTC timestamps, but running the code above generates timestamps in the local timezone of the user. We've been getting away with that inaccuracy so far, but now that we're providing an "official" wrapper, it should be done properly.

---

As a concrete suggestion how to achieve this
```suggestion
[tasks.timestamp]
cmd = "python -c \"import pandas; print(f'{pandas.Timestamp.now(tz=\"UTC\").timestamp():.0f}000')\""
description = "Generate a timestamp in milliseconds since epoch."
```

---

Thank you! I was not aware of this.